tern/extensions/scancode/executor.py

```diff
@@ -81,9 +81,11 @@ def filter_pkg_license(declared_license):
     it will represent the license as a string or list of strings.
     Given a scancode declared_license field, extract and return a
     license string'''
-    if isinstance(declared_license, dict):
+    if isinstance(declared_license, dict) and len(declared_license) != 0:
         try:
-            return declared_license['license']
```

Comment:
Do we need a try loop for this? We check if `declared_license` is a dict and there are more than 1 values for it, then we check if either `license` or `License` is a key in this dictionary.
Why not use the try loop for checking if there is a key called `license` or `License` and if neither one exists use `finally`?